drivers/volume/portworx/portworx.go

```diff
@@ -4753,3 +4771,50 @@ func (d *portworx) UpdatePoolLabels(n node.Node, poolID string, labels map[strin
 	return nil
 
 }
+
+// GetPoolIOPriorityStatus returns IO Priority of the Pool
+func (d *portworx) GetPoolLabelValue(poolUUID string, label string) (string, error) {
+	/* e.x
+	1) d.GetPoolLabelValue(poolUUID, "iopriority")
+	2) d.GetPoolLabelValue(poolUUID, "beta.kubernetes.io/arch")
+	3) d.GetPoolLabelValue(poolUUID, "medium")
+	*/
+	var PropertyMatch string
+	PropertyMatch = ""
+	pools, err := d.ListStoragePools(metav1.LabelSelector{})
+	log.FailOnError(err, fmt.Sprintf("Listing storage pools failed"))
+	for _, eachPool := range pools {
+		if eachPool.Uuid == poolUUID {
+			PropertyMatch = eachPool.Labels[label]
+			break
+		}
+	}
+	if PropertyMatch == "" {
+		return "", errors.New(fmt.Sprintf("Failed to Get [%s] for PoolUUID [%v]", label, poolUUID))
+	}
+	return PropertyMatch, nil
+}
+
+// IsNodeInMaintenance returns true if Node in Maintenance
+func (d *portworx) IsNodeInMaintenance(n node.Node) bool {
+	stNode, err := d.GetDriverNode(&n)
+	log.FailOnError(err, "Failed fetching Driver Details from Node [%v]", n)
+
+	if stNode.Status == api.Status_STATUS_MAINTENANCE {
+		return true
+	}
+
+	return false
+}
+
+// IsNodeOutOfMaintenance returns true if Node in Maintenance
+func (d *portworx) IsNodeOutOfMaintenance(n node.Node) bool {
+	stNode, err := d.GetDriverNode(&n)
+	log.FailOnError(err, "Failed fetching Driver Details from Node [%v]", n)
```

Comment:
Same here, don't print the whole node object here, just `node.Name` is enough

Reply:
done